tests/benchmarks/tpch/test_comparison_single_vm.py

```diff
@@ -0,0 +1,142 @@
+import dask
```

Comment:
Is this file still useful, or are we able to remove it?

Reply:
Probably, Polars/DuckDB are already running single node, I suppose we'll have something similar for spark/dask to get them on single. Will remove it.